eva-server/src/test/java/uk/ac/ebi/eva/server/ws/RegionWSServerV2Test.java

```diff
@@ -123,6 +124,10 @@ private void testGetVariantsByRegionHelper(String testRegion, int expectedVarian
         ResponseEntity<String> response = restTemplate.getForEntity(url, String.class);
         assertEquals(status, response.getStatusCode());
 
+        if (status == HttpStatus.NO_CONTENT) {
+            assertNull(response.getBody());
+            return null;
+        }
```

Comment:
For increased clarity, because this completely breaks the method flow.
```suggestion
        }

```